internal/routes/routes.go

```diff
@@ -0,0 +1,13 @@
+package routes
+
+import (
+	"github.com/gin-gonic/gin"
+)
+
+func InitRoutes(router *gin.Engine) {
```

Comment:
We need somewhere in this a way to generate/host a swagger-representation of this API.